SPECS/qdox/qdox.spec

```diff
@@ -39,65 +41,79 @@ QDox is a parser for extracting class/interface/method definitions
 from source files complete with JavaDoc @tags. It is designed to be
 used by active code generators or documentation tools.
 
+%package javadoc
+Summary:        Javadoc for %{name}
+Group:          Development/Libraries/Java
+
+%description javadoc
+API docs for %{name}.
+
 %prep
-%setup -q -n %{name}-%{name}-%{version}
+%setup -q -n %{name}-%{version}
+cp %{SOURCE1} build.xml
+%patch 0 -p1
```

Comment:
Nit: wouldn't `%autosetup` take care of this?

Reply:
Changed to autosetp although manual file processing comands are still needed.